.gitignore

```
@@ -3,3 +3,9 @@ datasets/__pycache__/
 eval/__pycache__/
 models/__pycache__/
 ops/__pycache__/
+
+.github/
+datasets/data/archive/
```

Nit: Ignoring the folder itself is enough
```suggestion
datasets/
```

---

Ah, yes. Thanks (: